main.nf

```diff
@@ -46,16 +45,19 @@ def helpMessage() {
       --peptides_split_minchunksize [int]   Used in combination with '--peptides' or '--proteins': minimum number of peptides that should be written into one chunk. Default: 5000
 
     References                              If not specified in the configuration file or you wish to overwrite any of the references
-      --genome [str]                        Specifies the ensembl reference genome version (GRCh37, GRCh38) Default: GRCh37
+      --genome_version [str]                Specifies the ensembl reference genome version (GRCh37, GRCh38) Default: GRCh37
```

Comment:
Why did you change this, to avoid confusion with the genome references? Anyway, this should be added to the CHANGELOG then, see below.

Reply:
Yes, `--genome` is used for the igenomes in other pipelines, therefore I thought it would make sense to use a different name.